haproxy/files/grafana_influxdb.json

```diff
@@ -9,7 +9,7 @@
         "iconSize": 13,
         "lineColor": "rgba(255, 96, 96, 0.592157)",
         "name": "Status",
-        "query": "select title,tags,text from annotations where $timeFilter and cluster = 'haproxy'",
+        "query": "select title,tags,text from annotations where $timeFilter and cluster = 'haproxy-openstack'",
```

Comment:
No the field name is "cluster" in the annotation messages. I did not want to change the Lua code.